private/src/gst/gstavbvideosrc.cpp

```diff
@@ -97,6 +106,11 @@ gst_avbvideosrc_class_init(GstAvbVideoSrcClass * klass)
             g_param_spec_string("stream-name", "Stream name",
                 "Name of the stream", DEFAULT_STREAM_NAME,
                 (GParamFlags)(G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS)));
+
+    g_object_class_install_property(gobject_class, PROP_STREAM_TYPE,
+            g_param_spec_uint("stream-type", "Stream type",
```

Comment:
Why not enum? It would also make things really clear to any user running `gst-inspect`. There's a nice example here https://gstreamer.freedesktop.org/documentation/plugin-development/basics/args.html.

Reply:
Should look better now. Thx